### PR TITLE
Refactor arrays containing fields for tests

### PR DESCRIPTION
**Pull request by marijnkampf**
## Description of change

Refactor of tests for Investment project fields.

## Test instructions

Tests should run as normal.

## Screenshots

No changes.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [x] Has the branch been rebased to main?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

**Review by samuele-mattiuzzo — COMMENTED** [review]
I second @oliverjwroberts sentiment around the mix of constants and strings, I'd suggest consistency to be prioritised

File: test/functional/cypress/specs/investments/constants.js [hunk 1]
```diff
@@ -11,3 +11,179 @@ export const INVESTMENT_PROJECT_FORM_VALIDATION = {
...
+    name: 'estimated_land_date',
+    message: INVESTMENT_PROJECT_FORM_VALIDATION.ESTIMATED_LANDDATE,
+  },
+  REFERRAL_SOURCE_ACTIVITY: {
+    name: 'referral_source_activity',
+    message: INVESTMENT_PROJECT_FORM_VALIDATION.REFERRAL_SOURCE_ACTIVITY,
+  },
+  INVESTOR_TYPE: {
+    name: 'investor_type',
+    message: 'Select an investor type',
```
[the remark this answers is not in the document]
Comment (marijnkampf) [reply test/functional/cypress/specs/investments/constants.js:51]: If a value is used once it's a string, if it's used more than once it's a constant.

File: test/functional/cypress/specs/investments/constants.js [hunk 2]
```diff
@@ -11,3 +11,179 @@ export const INVESTMENT_PROJECT_FORM_VALIDATION = {
...
+    name: 'number_safeguarded_jobs',
+    message: 'Value for number of safeguarded jobs is required',
+  },
+  R_AND_D_BUDGET: {
+    name: 'r_and_d_budget',
+    message: 'Select whether project has budget for research and development',
+  },
+  TOTAL_INVESTMENT: {
+    name: 'total_investment',
+    message: '-total_investment',
```
[the remark this answers is not in the document]
Comment (marijnkampf) [reply test/functional/cypress/specs/investments/constants.js:141]: I'll get rid of this as it's a placeholder that didn't end up getting used.

File: test/functional/cypress/specs/investments/constants.js [hunk 3]
```diff
@@ -11,3 +11,179 @@ export const INVESTMENT_PROJECT_FORM_VALIDATION = {
...
+  FIELDS.NAME,
+  FIELDS.DESCRIPTION,
+  FIELDS.BUSINESS_ACTIVITIES,
+  FIELDS.CLIENT_CONTACTS,
+  FIELDS.IS_REFERRAL_SOURCE,
+  FIELDS.ESTIMATED_LAND_DATE,
+  FIELDS.REFERRAL_SOURCE_ACTIVITY,
+]
+
+export const UPDATE_SUMMARY_ADDITIONAL_FIELDS = [
```
[the remark this answers is not in the document]
Comment (marijnkampf) [reply test/functional/cypress/specs/investments/constants.js:157]: That could be argued I suppose. I choose to make it more explicit in the `project-edit-stages-specs.js` file to show that the base fields are there for all stages and the additional fields differ for some stages.

[the remark this answers is not in the document]
Comment (marijnkampf) [reply test/functional/cypress/specs/investments/constants.js:153]: I've refactored this to be consistent.

**Review by samuele-mattiuzzo — APPROVED** [review]
So much red. I like it.

I'll let @oliverjwroberts resolve the conversations once he's happy with this too

**Review by oliverjwroberts — APPROVED** [review]

**Pull request merged**